numpy/testing/_private/utils.py

```diff
@@ -1508,6 +1515,12 @@ def assert_allclose(actual, desired, rtol=1e-7, atol=0, equal_nan=True,
     --------
     assert_array_almost_equal_nulp, assert_array_max_ulp
 
+    Notes
+    -----
+    When one of `actual` and `desired` is a scalar and the other is array_like,
```

Comment:
Use double backtick around parameter names to ensure sphinx won't try to make these links.
```suggestion
    When one of ``actual`` and ``desired`` is a scalar and the other is array_like,
```

Reply:
Thanks! I added the double backticks and adjusted the line lengths to within limits.

Reply:
Thanks @charris! I was only thinking about the warnings in nitpick sphinx mode and didn't know about this part of the standard.